src/main/java/core/basesyntax/AppRun.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class AppRun {
+    public static void main(String [] arg) {
+        FigureSupplier figSup = new FigureSupplier();
+        Figure [] figure = new Figure[new Random().nextInt(4) + 3];
```

Comment:
```suggestion
        Figure [] figure = new Figure[new Random().nextInt(4)];
```
Let's make 4 a constant 
https://mate-academy.github.io/jv-program-common-mistakes/java-core/abstract-class-interface/oop-advanced#all-magic-numbers-in-your-code-should-be-constants